### ISSUE TITLE
make it easier to remap short cuts for actions

### ISSUE THREAD

**Issue by kbilsted** [notepad-plus-plus/notepad-plus-plus#1879]
the current window "settings -> shortcut mapper" could use a text search field acting as a filter only showing matching strings. It is really laborious to find anything in the present window. Perhaps also a search for entries with a know shortcut would be nice

**Comment by dail8859** [comment]
The whole Shortcut Mapper could use an overhaul. [Here](https://notepad-plus-plus.org/community/topic/5/shortcuts) is a short forum discussion from about a year ago.

**Comment by lecramm** [comment]
support for multi key would be nice too, e.g. ctrl-R-W to show/hide white space/tabs. Standard this is ctrl-shift-W, but this closes notepad++.

**Comment by MetaChuh** [comment]
please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your support and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)

**Comment by kbilsted** [comment]
Why is this issue closed? nothing has been fixed in v7.6.6 or older. And no wording has been asked to be clarified?

**Comment by sasumner** [comment]
@kblisted, a filtering capability has been added since the inception of this issue (not sure exactly when or what version, but for sure sometime since May of 2016). Perhaps that is why it was closed?
